portals/devportal/src/main/webapp/source/src/app/components/Notifications/DeleteNotifications.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from 'react';
```

Comment:
Let's add the License header here as well as to the other new classes which is missing this header.